### PR TITLE
📖 Update instructions for Scorecard badge to README

### PR DESCRIPTION
**Pull request by azeemshaikh38**
Adds instructions on how to add a Scorecard badge to README and how to use it.

### CODE REVIEW

**Review by naveensrinivasan — APPROVED** [review]
Thank you!

File: README.md [hunk 1]
```diff
@@ -103,7 +105,25 @@ Then click "Add More Scanning Tools."
...
+(https://api.securityscorecards.dev/projects/github.com/{org}/{repo})
+```
+
+Once this badge is added, clicking on the badge will take users to the latest run result of Scorecard.
+
+![image](/images/badge.png)
+
+### Code Scanning Alerts
+
+A list of results is accessible by going in the Security tab and clicking "Code Scanning Alerts" (it can take a couple minutes for the run to complete and the results to show up). Click on the individual alerts for more information, including remediation instructions. You will need to click "Show more" to expand the full remediation instructions.
```
Comment (olivekl) [inline README.md:126]: Minor suggestion:
"Results are available a few minutes after the scan begins. Under the Security tab, choose "Code Scanning Alerts." Click on individual alerts for more information... (etc)"

File: README.md [hunk 2]
```diff
@@ -103,7 +105,25 @@ Then click "Add More Scanning Tools."
...
+
+Starting with scorecard-action:v2, users can add a Scorecard Badge to their README to display the latest status of their Scorecard results. This requires setting `publish_results: true` for the action and enabling `id-token: write` permission for the job (needed to access GitHub OIDC token). The badge is updated on every run of scorecard-action and points to the latest result. To add a badge to your README, copy and paste the below lines:
+
+```
+[![OpenSSF Scorecard]
+(https://api.securityscorecards.dev/projects/github.com/{org}/{repo}/badge)]
+(https://api.securityscorecards.dev/projects/github.com/{org}/{repo})
+```
+
+Once this badge is added, clicking on the badge will take users to the latest run result of Scorecard.
```
Comment (olivekl) [inline README.md:120]: Where is this latest run result? I.e., it will take you to the Code Scanning Alert for the repo?

**Review by olivekl — COMMENTED** [review]
LGTM, minor comments.

**Pull request merged**